content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -50,33 +55,43 @@ Use a single-node etcd cluster only for testing purpose.
 
 1. Run the following:
 
-    ```sh
-    ./etcd --listen-client-urls=http://$PRIVATE_IP:2379 --advertise-client-urls=http://$PRIVATE_IP:2379
-    ```
+   ```sh
+   etcd --listen-client-urls=http://$PRIVATE_IP:2379 \
```

Comment:
Hi @tengqm .
What are the benefits of reducing the number of characters per line (readability, easier to translate)?

Reply:
The benefit is about readability, e.g. number of arguments passed.
Readability is also improved when you are reading the doc on a pad.
It is not related to translation.